app/components/dashboard_component/dashboard_component.html.slim

```diff
@@ -10,7 +10,7 @@
       span.govuk-caption-l = @organisation.name
       h1.govuk-heading-l = t("jobs.dashboard.#{@selected_type}.with_count", count: @vacancies.count)
     .govuk-grid-column-one-quarter
-      = govuk_button_to t("buttons.create_job"), organisation_jobs_path, class: "float-right"
+      = govuk_button_to t("buttons.create_job"), organisation_jobs_start_path, method: :get, class: "float-right"
```

Comment:
you could use a link with the class `govuk-botton` here as it is just to get to the start page

Reply:
Ah good idea, thank you!